elements/pfelement/src/pfelement.js

```diff
@@ -287,6 +287,8 @@ class PFElement extends HTMLElement {
   }
 
   resetContext(fallback) {
+    if (this.isIE11) return;
```

Comment:
In debugging I saw that the resetContext was rerunning several times but since our surface components default to light context, there really wasn't a need to update the context